Pipelines/Scripts/validatecode.ps1

```diff
@@ -281,7 +281,8 @@ function GetProjectRelativePath {
 # (which run serially) causes noticable delays in responsiveness in the Unity editor.
 $InitializeOnLoadExceptions = [System.Collections.Generic.HashSet[String]]@(
     "com.microsoft.mrtk.core/Editor/MRTKSettings.cs",
-    "com.microsoft.mrtk.core/Editor/EditorProjectUtilities.cs"
+    "com.microsoft.mrtk.core/Editor/EditorProjectUtilities.cs",
+    "com.microsoft.mrtk.core/Editor/MRTKProjectValidation.cs"
```

Comment:
Do we need this as part of CI? When we work on the template project, we will see the guidance.

Reply:
Kurtis added the CI check for [InitializeOnLoadMethod] so yes we need to add this file to the allow list for it to pass CI.